src/main/resources/schema.sql

```diff
@@ -0,0 +1,11 @@
+CREATE SCHEMA IF NOT EXISTS `mtgadb`;
+USE `mtgadb`;
+
+-- -----------------------------------------------------
+-- Table `mtgadb`.`klient`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `mtgadb`.`klient` (
```

Comment:
Tak jak pisałem w kontekście modelu ja bym proponował w kodzie używać tylko angielskich nazw (jest to raczej ogólnie przyjęty standard - no może z wyjątkiem kodu dla polskich instytucji publicznych)